switchmap/poller/snmp/mib/generic/mib_lldp.py

```
@@ -95,32 +104,38 @@ def layer1(self):
+            "lldpRemSysDesc",
+            "lldpRemPortDesc",
+            "lldpRemSysCapEnabled",
+        ]
+        final = defaultdict(lambda: defaultdict(dict))
 
-        # Get interface lldpRemPortDesc data
-        values = self.lldpremportdesc()
-        if values is not None:
-            for key, value in values.items():
-                final[key]["lldpRemPortDesc"] = value
+        for i, (method_name, values) in enumerate(zip(method_names, results)):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)